recipes/trefide/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "trefide" %}
+{% set version = "1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jw3132/conda-forge-tar/raw/master/trefide-1.2.tar.gz
```

This doesn't look like an official source. What is the reason for mirroring to GitHub?

The URL should always depend on the version to avoid mistakes when updating:
```suggestion
  url: https://github.com/jw3132/conda-forge-tar/raw/master/trefide-{{ version }}.tar.gz
```

---

The version variable issue has been fixed.

Our code is stored in github. Since we need to provide a tar ball to conda-forge, so we created a tar ball for this purpose.

Thanks.

Jian